src/hooks/media.hook.ts

```diff
@@ -1,15 +1,22 @@
 import { Breakpoints } from './../App.constants';
 // Helper functions to determine screen size in JS
 import { useMedia } from 'use-media';
+import { detect } from 'detect-browser';
+
+const isIE11 = detect()!.name === 'ie';
+
+export function useIE11() {
```

Comment:
There seems to be no need to put the check into a function

Reply:
```
var ua = window.navigator.userAgent;
var isIE = /MSIE|Trident/.test(ua);

if ( isIE ) {
  //IE specific code goes here
}
```

Reply:
True